src/sdg/accounts/views/password_reset.py

```diff
@@ -8,3 +14,19 @@ class PasswordResetView(IPThrottleMixin, auth_views.PasswordResetView):
     throttle_visits = 5
     throttle_period = 60
     throttle_methods = ["get"]
+
+
+@staff_member_required
+def ResendInventation(request):
```

Comment:
def resent_invitation(request) 

If it isn't a class, don't pretend it is one. Or switch to CBV instead.

Reply:
Please use a class (Class Based View - CBV)